src/main/resources/edu/cornell/kfs/module/ld/document/datadictionary/CuYearEndSalaryExpenseTransferDocument.xml

```diff
@@ -42,4 +57,14 @@
     </property>
   </bean>
 
+  <bean id="YearEndSalaryExpenseTransferDocument-workflowAttributes" parent="YearEndSalaryExpenseTransferDocument-workflowAttributes-parentBean">
```

Comment:
For consistency, lets rename this to CuYearEndSalaryExpenseTransferDocument and we will need to add it workflow attributes in CuYearEndSalaryExpenseTransferDocument.  Double check we don't need to worry about XML ingestion, I don't think we do.